.sandbox/guacamole-jetty/rdp.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+function resource_output() {
+    jq -cMr ".$1.value" </run/sandbox/fs/resources/windows/state 2>/dev/null || true
+}
+
+function wait_for_resource() {
+    while true ; do
+        [[ -z "$(resource_output public_dns)" ]] || return 0
```

Comment:
This hard-coded piece restricted the Windows instance must be accessible using a public IP. As a general solution, private instances should also be supported.